app_utils/sections/settings.py

```diff
@@ -143,6 +144,22 @@ async def settings(q: Q) -> None:
                 ]
             ),
             ui.separator("Default Experiment Settings"),
+            ui.inline(
+                items=[
+                    ui.label("Gpu used for Chat", width=label_width),
+                    ui.spinbox(
+                        name="gpu_used_for_chat",
+                        label=None,
+                        min=1,
+                        max=torch.cuda.device_count(),
+                        step=1,
+                        value=q.client["gpu_used_for_chat"],
+                        width=textbox_width,
+                        trigger=False,
+                        tooltip="Set the gpu id that is used for the chat window.",
+                    ),
+                ]
+            ),
```

Comment:
maybe another separator: "Chat Settings" ?

Reply:
Good idea, changed it